packages/cli/src/commands/releasegold/set-beneficiary.ts

```diff
@@ -12,38 +13,52 @@ export default class SetBeneficiary extends ReleaseGoldCommand {
       required: true,
       description: 'Address of the new beneficiary',
     }),
+    yesreally: flags.boolean({
+      description: 'Override prompt to set new beneficiary (be careful!)',
+    }),
   }
 
   static args = []
 
   static examples = [
-    'set-beneficiary --contract 0x5409ED021D9299bf6814279A6A1411A7e866A631 --new-beneficiary 0x6Ecbe1DB9EF729CBe972C83Fb886247691Fb6beb',
+    'set-beneficiary --contract 0x5409ED021D9299bf6814279A6A1411A7e866A631 --new_beneficiary 0x6Ecbe1DB9EF729CBe972C83Fb886247691Fb6beb',
   ]
 
   async run() {
     // tslint:disable-next-line
     const { flags } = this.parse(SetBeneficiary)
     const newBeneficiary = flags.new_beneficiary
-    const response = await prompts({
-      type: 'confirm',
-      name: 'confirmation',
-      message:
-        "Are you sure you want to set a new beneficiary? This will forfeit the current beneficiary's controls. (y/n)",
-    })
+    if (!flags.yesreally) {
+      const response = await prompts({
+        type: 'confirm',
+        name: 'confirmation',
+        message:
+          "Are you sure you want to set a new beneficiary? This will forfeit the current beneficiary's controls. (y/n)",
+      })
 
-    if (!response.confirmation) {
-      console.info('Aborting due to user response')
-      process.exit(0)
+      if (!response.confirmation) {
+        console.info('Aborting due to user response')
+        process.exit(0)
+      }
     }
+    // Multisig is the owner of the ReleaseGold contract
     const owner = await this.releaseGoldWrapper.getOwner()
     const releaseGoldMultiSig = await this.kit.contracts.getMultiSig(owner)
+    const releaseOwner = await this.releaseGoldWrapper.getReleaseOwner()
+    const beneficiary = await this.releaseGoldWrapper.getBeneficiary()
+    console.log(releaseOwner, beneficiary)
     const tx = this.releaseGoldWrapper.setBeneficiary(newBeneficiary)
-    const multiSigTx = await releaseGoldMultiSig.submitOrConfirmTransaction(
+    const multiSigTxReleaseOwner = await releaseGoldMultiSig.submitOrConfirmTransaction(
+      this.contractAddress,
+      tx.txo
+    )
+    await displaySendTx<any>('setBeneficiaryOneOfTwo', multiSigTxReleaseOwner, {
+      from: releaseOwner,
```

Comment:
In what circumstances would you expect the user to have both keys accessible? IMO this command should take a `--from` flag, add a "check" that the "from" flag is a signatory on the multisig, and then submit or confirm a single multisig tx from that "from" address